test/end-to-end/cases/force-focus_none.xspec

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- Remove focus from all the scenarios -->
+<?xspec-test force-focus=#none?>
+
+<x:description query="x-urn:test:do-nothing" query-at="../../do-nothing.xqm"
+	schematron="../../do-nothing.sch" stylesheet="../../do-nothing.xsl"
+	xmlns:t="http://www.jenitennison.com/xslt/xspec"
```

Comment:
Is it necessary to bind the `t` prefix when this file doesn't use it directly? The test still passed when I removed this line or changed it to bind the prefix to some other URI.

Reply:
I had the same impression and removed `xmlns:t` initially. However, removing it led to a crash in the XQuery test. It's worth noting that the XSLT compiler offers [a finer namespace treatment](https://github.com/xspec/xspec/blob/f24e6971267e64bcabb035f78eb71f5a1ba5869b/src/compiler/xslt/compile/compile-expect.xsl#L87-L89), which is lacking in the XQuery compiler.

Reply:
Sorry for the distraction. Now I'm seeing the static error in the compiled query when this declaration isn't in the test. I'm not sure why it seemed to pass when I made my earlier comment; my mistake.